frontend/cypress/integration/multiProject.spec.js

```diff
@@ -0,0 +1,291 @@
+import faker from 'faker'
+const syncedRepoUrlMultiProjects =
+  'https://github.com/kitspace-forks/DIY_particle_detector'
+const syncedRepoUrl = 'https://github.com/kitspace-forks/CH330_Hardware'
+const multiProjectsNames = ['alpha-spectrometer', 'electron-detector']
+const multiProjectsRepoName = 'DIY_particle_detector'
+const normalRepoName = 'CH330_Hardware'
+
+describe('Render project cards', () => {
+  before(() => {
+    /*
+     * The purpose of this isn't actually visiting the homepage.
+     * Sometimes, the frontend has a slow startup time which results in a random failure.
+     */
+    cy.visit('/')
+  })
+
+  it('should render a card for each multiproject', () => {
+    const username = faker.name.firstName()
+    const email = faker.internet.email()
+    const password = '123456'
+
+    const repoName = syncedRepoUrlMultiProjects.split('/').slice(-1).toString()
+
+    cy.intercept('http://gitea.kitspace.test:3000/user/kitspace/**').as('sign_in')
+
+    cy.createUser(username, email, password)
+    cy.visit('/login')
+    cy.signIn(username, password)
+    cy.wait('@sign_in')
+
+    cy.visit('/projects/new')
+
+    cy.url().then(url => {
+      if (!url.endsWith('/projects/new')) {
+        cy.visit('/projects/new')
+      }
+    })
```

Comment:
If I'm reading this right this essentially forces a refresh, this shouldn't be necessary.

Reply:
Oops, missed a `!`, never mind.

Reply:
Sometimes `visit('projects/new')` doesn't redirect to the page in the next steps executed on the homepage not in `projects/new`. This won't trigger a redirect if the `visit('projects/new') redirection was successful,